agenta-web/src/components/AppSelector/modals/CreateAppStatusModal.tsx

```diff
@@ -112,7 +112,7 @@ const CreateAppStatusModal: React.FC<Props & React.ComponentProps<typeof Modal>>
                         },
                     }
                     if (obj.starting_app?.type === "loading") obj.starting_app.type = "success"
-                    if (appId) router.push(`/apps/${appId}/playground`)
```

Comment:
Let's keep this to playground. I think it makes sense to navigate the user to the playground on app creation